controllers/k8ssandra/stargate.go

```diff
@@ -105,7 +105,18 @@ func (r *K8ssandraClusterReconciler) reconcileStargate(
 }
 
 // TODO move to stargate package
-func (r *K8ssandraClusterReconciler) newStargate(stargateKey types.NamespacedName, kc *api.K8ssandraCluster, stargateTemplate *stargateapi.StargateDatacenterTemplate, actualDc *cassdcapi.CassandraDatacenter) *stargateapi.Stargate {
+func (r *K8ssandraClusterReconciler) newStargate(stargateKey types.NamespacedName, kc *api.K8ssandraCluster, stargateTemplate *stargateapi.StargateDatacenterTemplate, actualDc *cassdcapi.CassandraDatacenter, dcTemplate api.CassandraDatacenterTemplate) *stargateapi.Stargate {
+	cassandraEncryption := stargateapi.CassandraEncryption{}
+	if dcTemplate.CassandraConfig != nil {
+		if dcTemplate.CassandraConfig.CassandraYaml.ClientEncryptionOptions != nil && kc.Spec.Cassandra.ClientEncryptionStores != nil {
+			cassandraEncryption.ClientEncryptionStores = kc.Spec.Cassandra.ClientEncryptionStores
+		}
+
+		if dcTemplate.CassandraConfig.CassandraYaml.ServerEncryptionOptions != nil && kc.Spec.Cassandra.ServerEncryptionStores != nil {
```

Comment:
Same here, what if encryption is `none`?

Reply:
switched to using `ServerEncryptionEnabled`